app/deliver_grant_funding/routes.py

```
@@ -180,6 +182,30 @@ def list_grants() -> Response | str:
     return render_template("deliver_grant_funding/grant_list.html", grants=grants)
 
 
+@deliver_grant_funding_blueprint.route("/users/<uuid:grant_id>", methods=["GET", "POST"])
+@mhclg_login_required
+def list_users_for_grant(grant_id: UUID) -> str:
+    grant = interfaces.grants.get_grant(grant_id)
```

Will this ever be `None` if we're passing a (hopefully) valid UUID through? The `get_grants()` interface also just uses `get_one` so should raise an exception if there isn't just one found (ie. returns None or more than one).

---

I don't know why we expect that we getting multiple grants over here so changed the DB query 

`return db.session.get(Grant, grant_id)`

this returns None